src/EnergyPlus/UnitarySystem.cc

```diff
@@ -12662,7 +12676,6 @@ namespace UnitarySystems {
                 NoLoadHumRatOut = state.dataLoopNodes->Node(OutletNode).HumRat;
 
                 Real64 NoOutput = 0.0; // CoilSystem:Cooling:DX
-                Real64 ReqOutput = 0.0;
```

Comment:
This fixes the cooling coil setpoint control operation (this line creates a shadow of `ReqOutput` which masks the value set up in [line 12438](https://github.com/NREL/EnergyPlus/blob/197358237f1faf861ea76ba458d05bf057e4436d/src/EnergyPlus/UnitarySystem.cc#L12438-L12442).

Reply:
Agreed